pkg/network/setup/netstat_test.go

```diff
@@ -463,6 +463,53 @@ var _ = Describe("netstat", func() {
 		})
 	})
 
+	Describe("primary interface position in spec", func() {
+		const (
+			prNetworkName   = "primary"
+			prMAC           = "1C:CE:C0:01:BE:E7"
+			secNetworkName1 = "secondary1"
+			secMAC1         = "1C:CE:C0:01:BE:E9"
+			secNetworkName2 = "secondary2"
+			secMAC2         = "1C:CE:C0:01:BE:E8"
+		)
+
+		BeforeEach(func() {
+			setup.Vmi.Spec.Networks = append(setup.Vmi.Spec.Networks, newVMISpecPodNetwork(prNetworkName), newVMISpecMultusNetwork(secNetworkName1))
+		})
+
+		DescribeTable("verify primary interface is always first in Status.Interfaces list", func(vmiSpecIfaces []v1.Interface, domainIfaces []api.Interface, statusIfaces []v1.VirtualMachineInstanceNetworkInterface) {
```

Comment:
I think `statusIfaces` is the expected result, right? It is hard to understand it based on the name.

Reply:
name updated